lua/galaxyline/provider_diagnostic.lua

```diff
@@ -31,6 +31,16 @@ local function diagnostic_coc_hint()
   return ''
 end
 
+-- coc info
+local function diagnostic_coc_info()
+  local has_info,info = pcall(vim.fn.nvim_buf_get_var,0,'coc_diagnostic_info')
+  if not has_info then return end
+  if info.information > 0 then
+    return  info.information
```

Comment:
It seems that we wrote a lot of repetitive code, we only need to judge the size of different indexes, right? Maybe it can be solved with a function or wrap it with design patterns